### ISSUE TITLE
Doesn't work?

### ISSUE THREAD

**Issue by pwFoo** [fiatjaf/zepto-backstretch#1]
Tested your version with latest zepto and also the linked one `http://cdnjs.cloudflare.com/ajax/libs/zepto/1.0/zepto.min.js`. Is there a fixed version available?

Would like to test your additional features ;)

**Comment by fiatjaf** [comment]
Oh, I'm sorry, I don't use this anymore, so I don't know. It used to work in the past.

It probably works with jQuery? (I just tried this Zepto approach to see if anyone else would be interested).

There was someone some time ago who merged some of my changes here into an alternative jquery-backstretch fork. This person merged many PRs that were forgotten and unmerged on jquery-backstretch and created a kind of super-fork. But since he didn't merge the PR as it was and through GitHub, it doesn't appear in the graph.

Maybe it is https://github.com/danielgindi/jquery-backstretch

**Comment by fiatjaf** [comment]
I will close this repo so I'll stop misleading people.

**Comment by pwFoo** [comment]
Thanks for come back to answer my question, @fiatjaf :)

But the super-fork misses your alternative image size feature? That could be interesting because of responsive designs and mobile devices...

**Comment by pwFoo** [comment]
Ok, should be **Automatic resolution selection**, but minimized file size increaed to 16kb (yours: 6kb, original 4kb). 
I'm happy with srobbin version at the moment, but Automatic resolution selection sounds like a great improvement to me. So I'll ask there because of zepto support which is much smaller than jquery... ;)

**Comment by pwFoo** [comment]
Your plugin works with jquery, but doesn' work if I replace jquery with zepto, but maybe it isn't your plugins problem.
So maybe I try your extra features with jquery first and take a closer look at the zepto compatibility later...

**Comment by fiatjaf** [comment]
Yes, there's a [Zepto module](http://zeptojs.com/#modules) that is needed. I don't remember what is it, and I just discovered this later (I'm not much a Zepto user).

If you are willing to try and tweak this, I suggest you try the `fx` module, but I'm not sure it is it. Perhaps the best solution is to keep using jQuery, unless you have a hard Zepto requirement.

I'm sorry I'm not helping you more.

**Comment by fiatjaf** [comment]
Thank you using this fork. Maybe we should work to bring the automatic resolution selection feature to the super fork and only after that delete this repository.

**Comment by pwFoo** [comment]
But this repo is also nice to have... It's a smaller version of the super fork...
